eclipse-collections/src/main/java/org/eclipse/collections/impl/map/mutable/UnifiedMap.java

```diff
@@ -1515,7 +1515,8 @@ public void readExternal(ObjectInput in) throws IOException, ClassNotFoundExcept
     {
         int size = in.readInt();
         this.loadFactor = in.readFloat();
-        this.init(Math.max((int) (size / this.loadFactor) + 1,
+        this.init(Math.max(
```

Comment:
should be `Math.max` be in a new line?

Reply:
I don't think so. init() only has a single parameter. Imagine extracting it as local variable.

```java
        int initialCapacity = Math.max(
                (int) (size / this.loadFactor) + 1,
                DEFAULT_INITIAL_CAPACITY);
        this.init(initialCapacity);
```

vs

```java
        this.init(
                initialCapacity);
```